### PR TITLE
[TESTED] added yakuza

### PR DESCRIPTION
**Pull request by ChaoticNeutralGnome**

### CODE REVIEW

File: Games/types/Mafia/roles/cards/CommitSeppuku.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const Card = require("../../Card");
+const { PRIORITY_KILL_DEFAULT } = require("../../const/Priority");
+
+module.exports = class CommitSeppuku extends Card {
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/CommitSeppuku.js:4]: this made me laugh

File: Games/types/Mafia/roles/cards/CommitSeppuku.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        super(role);
+
+        this.meetings = {
+            "Convert Player": {
+                states: ["Night"],
+                flags: ["voting"],
+                targets: { include: ["alive"], exclude: ["Mafia"] },
+                action: {
+                    labels: ["kill", "seppuku"],
+                    priority: PRIORITY_KILL_DEFAULT,
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/CommitSeppuku.js:16]: this should be a conversion priority not a kill priority imo

what is the intended mechanic if someone is masoned, culted, yakked the same night?

**Comment by pikulet** [comment]
**yakuza tests**
- [ ] target is masoned, culted and yakked same night --> i think yak should take priority?
- [ ] yak can be saved on yak night, giving him more yak chances

**Comment by pikulet** [comment]
image

the target was masoned and yakked same night
the headmason (buyle) went on him and died from mafia mismason

this is a mason bug that surfaced due to yakuza.
Priority -2 --> target is masoned
priority 0 --> target is yakked
priority -1 --> check for mafia mismason

**Comment by ChaoticNeutralGnome** [comment]
I agree that this is probably a bug with mason and not yakuza. Possibly if the creator didn't want death to happen at that priority they could queue a later death action, kind of like how KillLycanVisitors does it?

File: Games/types/Mafia/roles/cards/CommitSeppuku.js (hunk 2 above)
Comment (ChaoticNeutralGnome) [reply Games/types/Mafia/roles/cards/CommitSeppuku.js:16]: No specific intended order. I'm not sure what happened in em or what we want to happen.

**Comment by pikulet** [comment]
lgtm, the mason bug is filed as #198

**Pull request merged**